wiki/osu!tourney/en.md

```diff
@@ -332,34 +327,33 @@ Rename or delete the `tournament.cfg` file.
 
 ### My client is not spectating!
 
-**Make sure you have joined the room!**
+Make sure you have **joined the room!**
 
 Click the room name on the bottom black panel until it is **bold**.
 If the client is still not spectating, click the `Panic` button.
 
 ### The rooms are not showing up!
 
-- Your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
-- Make sure you use the correct room name (see the **Match Creation** section).
-  Renaming an existing multiplayer room will not work.
-- Recreate the entire room, if you did something wrong.
+It is possible that your multiplayer room is named incorrectly or you used the wrong acronym in the `tournament.cfg`.
+Make sure you are using the correct room name (see the **Match Creation** section), renaming an existing multiplayer room will not work.
```

Comment:
A hyphen would.l serve the same purpose as the comma here but i prefer the comma personally.